eng/tools/generator/config/validate/localValidator.go

```diff
@@ -86,9 +86,26 @@ func getReadmeGoFromReadme(readme string) string {
 	return strings.ReplaceAll(readme, readmeFilename, goReadmeFilename)
 }
 
+func findModuleName(content []byte) bool {
+	return regexp.MustCompile(goReadmeModuleName).Match(content)
+}
+
+func GetModuleName(content []byte) (string, string) {
+	if findModuleName(content) {
```

Comment:
If `findModuleName` only used for `GetModuleName`, could you please merge the logic?

Reply:
merged